packages/yarnpkg-core/sources/Configuration.ts

```diff
@@ -1790,7 +1790,10 @@ export class Configuration {
     return {os, cpu, libc};
   }
 
-  isInteractive({interactive, stdout}: {interactive?: boolean, stdout: Writable}): boolean {
+  isInteractive({interactive, stdin, stdout}: {interactive?: boolean, stdin: Readable, stdout: Writable}): boolean {
```

Comment:
This is a breaking change.

Reply:
:thinking: Indeed, didn't realize 4.4.0 was released with the previous PR.

Given the fact that GH code search doesn't return anything for `configuration.isInteractive` and the change has only been out for one week, I'd say there's about a 0% chance that anybody is using it and would prefer to land this change before anybody gets the chance to use it.

Reply:
It's probably fine but could we make it optional with a TODO note to fix it for the next major?